.codacy.yml

```diff
@@ -0,0 +1,3 @@
+---
+exclude_paths:
```

Comment:
Can you add a comment explaining to people why this folder should be ignored?

(it's because it's autogenerated code right?)

Reply:
Will do! (yes)

```suggestion
exclude_paths:
  # The "projects" directory is autogenerated by the "generate-rest-client.ps1" script
  # No need to verify the code standards there as we don't strictly control that code
```